media/com_languages/js/admin-override-edit-refresh-searchstring.es6.js

```diff
@@ -6,7 +6,7 @@
 document.addEventListener('DOMContentLoaded', () => {
   document.getElementById('jform_searchstring').addEventListener('focus', (event) => {
     if (!Joomla.overrider.states.refreshed) {
-      const expired = Joomla.getOptions('search-string-expired');
+      const expired = document.getElementById('overrider-spinner').dataset.searchstringexpired;
```

Comment:
Please don't use `dataset.searchstringexpired`, but instead, getAttribute('data-searchstringexpired'). It's much faster